TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/preference/ThemeFragment.java

```diff
@@ -88,7 +91,7 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
     }
 
     private void changeTheme() {
-        // For initializing custom colors
+        // Init custom colors
         switch (PrefTheme.getTheme()) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1301)